packages/accounts/src/light-account/accounts/base.ts

```diff
@@ -161,26 +167,36 @@ export async function createLightAccountBase({
     });
   };
 
-  const signWith1271Wrapper = async (msg: Hex): Promise<`0x${string}`> => {
+  const signWith1271WrapperV1 = async (hashedMessage: Hex): Promise<Hex> => {
     return signer.signTypedData({
       // EIP712Domain(string name,string version,uint256 chainId,address verifyingContract)
       // https://github.com/alchemyplatform/light-account/blob/main/src/LightAccount.sol#L236
       domain: {
         chainId: Number(client.chain.id),
         name: type,
         verifyingContract: accountAddress,
-        version: version === "v2.0.0" ? "2" : "1",
+        version: "1",
       },
       types: {
         LightAccountMessage: [{ name: "message", type: "bytes" }],
       },
       message: {
-        message: msg,
+        message: hashedMessage,
       },
       primaryType: "LightAccountMessage",
     });
   };
 
+  const { signMessage: signMessageV2, signTypedData: signTypedDataV2 } =
+    getErc1271SigningFunctions({
+      accountAddress,
+      accountName: type,
+      accountVersion: "2",
+      chainId: chain.id,
+      signer,
+      wrapperTypeName: "LightAccountMessage",
```

Comment:
If it simplifies things, this can also be fixed to something like "ERC1271Wrapper" across all accounts that the SDK supports.

Reply:
Yea, that's the string that frangio uses in his example code.

I thought I'd go with `LightAccountMessage` here for two reasons:

1. The encoding fails if the wrapper type shares a name with a type in the payload. If all accounts using this encoding use the same wrapper type name, it makes it more likely that there will be a type name conflict at some point if someone needs to send a "meta" message containing a similarly encoded message.
2. This name will be displayed to end users in their wallet software. I thought "LightAccountMessage" is a little more friendly and also clearer about where this encoding is coming from.

But I don't feel too strongly about this. It doesn't really simplify things either way, we can make this string whatever we want.